app/models/data/returning_users.rb

```diff
@@ -5,22 +5,16 @@ class ReturningUsers
     class << self
       # @return [Array<String>]
       def column_names
-        [
-          'Weekly Returning Users',
-          'Monthly Returning Users',
-          'Bimonthly Returning Users',
-          'Quarterly Returning Users',
-        ]
+        ['Weekly Returning Users', 'Monthly Returning Users', 'Quarterly Returning Users']
```

Comment:
Similarly annoying format comment, but personally I think formatting our headings and rows on their own lines improves readability in the code and specs.